### DISCUSSION TITLE
disocrd F me, and now i cant rejoin the dicord server, can you helpo

### DISCUSSION THREAD

**Discussion by TamirRothschild** [Dadoum/Sideloader#18] [General]
i hate discord

**Comment by Dadoum** [comment]
Which problem have you encountered? Anyway doing support here is better as it could benefit to everyone.

**Reply by TamirRothschild** [reply]
its not sideloder just the whole discord server i lost everything there, all the altstore and linux related and jailbreak in general

**Reply by TamirRothschild** [reply]
btw the sideloader works great with the misaka, btw why did it install altstore as well as the ipa itself?

**Reply by Dadoum** [reply]
it shouldn't have, the software never even acknowledge the existence of AltStore

**Reply by TamirRothschild** [reply]
idk it did install it

**Reply by TamirRothschild** [reply]
can you send me the link to the discord server pls
to my email tamirrothschild1@gmail.com
